Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,39 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '@{{plainid}}\.'
```

Comment:
The `@` symbol should probably have the scope `punctuation.definition.annotation.scala`, like how variables do in PHP.